extension/copy-on-y.js

```diff
@@ -0,0 +1,16 @@
+/* globals utils */
+
+'use strict';
+
+const Y_KEYCODE = 89;
+
+window.enableCopyOnY = () => {
+	$(document).on('keyup', ({keyCode}) => {
+		if (keyCode === Y_KEYCODE) {
+			const commitIsh = $('.commit-tease-sha').text().trim();
+			const uri = location.href.replace(/\/blob\/(.+)\//, `/blob/${commitIsh}/`);
```

Comment:
A couple of things here:

1. Is there a reason you're taking the hash from the DOM instead of the `location`? It kind of threw me at first because it didn't look like the URL I was expecting.

2. The regex is too open and you're replacing part of the filename too. I think `/\/blob\/(\w+)\//`might be a better test. Try it here: https://github.com/sindresorhus/refined-github/blob/issue201/extension/content.js. It copies `https://github.com/sindresorhus/refined-github/blob/885f2c6/content.js` to the clipboard, but the URL says https://github.com/sindresorhus/refined-`github/blob/885f2c698d9e2695258127eb7f02553661b396d2/extension/content.js` (it took out `/extension`)